### PR TITLE
[update][s] Fixing Dial code problem - refs #87.

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made

- Added function to handle the Dominican multiple Dial codes
- Fixed dataset

## Reference to the issue:
#87 

cc @anuveyatsu

### CODE REVIEW

File: scripts/cleanup.py [hunk 1]
```diff
@@ -1,5 +1,14 @@
+import re
 import pandas as pd
 
+def format_dial_codes(dial):
```
Comment (anuveyatsu) [inline scripts/cleanup.py:4]: @gradedSystem what was the old script that handled this case correctly?

Comment (gradedSystem) [reply scripts/cleanup.py:4]: @anuveyatsu I believe the old script `scripts/statoids.py` should be handling this case:
https://github.com/datasets/country-codes/blob/92bbe1a42dd38503c43e29f8218d850748c29128/scripts/statoids.py#L82
however it doesn't seem to work in this case

Comment (anuveyatsu) [reply scripts/cleanup.py:4]: @gradedSystem ok so this is a unique case only for Dominican Republic. In this case I'm OK with the change you are proposing in this PR :+1:

**Review by anuveyatsu — APPROVED** [review]
LGTM

File: scripts/cleanup.py (hunk 1 above)
Comment (gradedSystem) [reply scripts/cleanup.py:4]: Yes it is the only case

**Pull request merged**